src/mame/drivers/viper.cpp

```diff
@@ -3424,14 +3424,14 @@ GAME(2001, jpark3u,   jpark3,    viper,     jpark3,     viper_state, init_viperc
 GAME(2001, mocapglf,  kviper,    viper_omz, mocapglf,   viper_state, init_vipercf,  ROT90, "Konami", "Mocap Golf (ver UAA)", MACHINE_NOT_WORKING)
 GAME(2001, mocapb,    kviper,    viper,     mocapb,     viper_state, init_vipercf,  ROT90, "Konami", "Mocap Boxing (ver AAB)", MACHINE_NOT_WORKING)
 GAME(2001, mocapbj,   mocapb,    viper,     mocapb,     viper_state, init_vipercf,  ROT90, "Konami", "Mocap Boxing (ver JAA)", MACHINE_NOT_WORKING)
-GAME(2001, p911,      kviper,    viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 911 (ver AAE)", MACHINE_NOT_WORKING)
-GAME(2001, p911k,     p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 911 (ver KAE)", MACHINE_NOT_WORKING)
-GAME(2001, p911ac,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 911 (ver AAC)", MACHINE_NOT_WORKING)
-GAME(2001, p911kc,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 911 (ver KAC)", MACHINE_NOT_WORKING)
-GAME(2001, p911ud,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 24/7 (ver UAD)", MACHINE_NOT_WORKING)
+GAME(2001, p911,      kviper,    viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "The Keisatsukan: Shinjuku 24-toki (ver AAE)", MACHINE_NOT_WORKING)
```

Comment:
Some can argue that this could be ji instead of toki but in recent days or months The Keisatsukan: Shinjuku 24-Toki sounds more appropriate for the Japanese title.

Reply:
It's not ambiguous here. When you're dealing with times, like in the game title here, it'll be "ji" instead of "toki". "24-toki" will always be wrong.

Reply:
According to Wikipedia (even though it’s not always the best) and a few sites it says Toki instead of ji.

Reply:
Then Wikipedia and all of those sites are wrong.

Watch the opening of the game. At 40s into the video when it gets to the title screen the announcer reads the game title and you can clearly hear it's "ji" and not "toki".

https://youtu.be/nIhGavncFtw?t=40

Reply:
@987123879113 is correct it is "ji". It cannot be "toki" in this grammatical context. Moreover, the voice over, in the PS2 version of the game at least, speaks the title.

Reply:
Alright, reverting the changes soon.